test/e2e/framework.go

```diff
@@ -140,9 +140,10 @@ const (
 	defaultCHDatabaseURL                = "tcp://clickhouse-clickhouse.flow-visibility.svc:9000"
 
 	statefulSetRestartAnnotationKey = "antrea-e2e/restartedAt"
-
-	iperfPort    = 5201
-	iperfSvcPort = 9999
+	randomPatchAnnotationKey        = "test.antrea.io/random-value"
+	annotationValueLen              = 8
```

Comment:
these constants should be moved to test/e2e/fqdn_dns_cache_test.go, at the right scope